matlab/ReadEclipseSummary.m

```diff
@@ -1,7 +1,255 @@
-function [ output_args ] = ReadEclipseSummary( input_args )
-%READECLIPSESUMMARY Summary of this function goes here
-%   Detailed explanation goes here
+function [ output_data ] = ReadEclipseSummary( name )
```

Comment:
Same as for ADGPRS reader: Describe what is returned, incuding the format. Also, state what the parameter should be, including the format, and gine an example.